ygg_entrypoint.sh

```diff
@@ -2,16 +2,13 @@
 
 set -ex
 
-mkdir -p /dev/net
-mknod /dev/net/tun c 10 200
-chmod 600 /dev/net/tun
-
-if [ ! -f "/etc/config.conf" ]; then
-  echo "generate /etc/config.conf"
-  yggdrasil --genconf > "/etc/config.conf"
+if [ ! -f "/etc/yggdrasil.conf" ]; then
+  echo "generating new configurations at /etc/yggdrasil.conf"
+  yggdrasil --genconf > "/etc/yggdrasil.conf"
+  sed -i "/Peers: \[\]/c\  Peers: \n  [\n    tls:\/\/54.37.137.221:11129\n  ]" /etc/yggdrasil.conf
```

Comment:
peers come and go, maybe a list of active peers can be generated instead from the public peers list. Or they can be passed as an env var to the chart.

Reply:
done as env variable